pkg/gds/store/trtl/trtl_test.go

```diff
@@ -0,0 +1,238 @@
+package trtl_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/suite"
+	store "github.com/trisacrypto/directory/pkg/gds/store/trtl"
+	"github.com/trisacrypto/directory/pkg/trtl"
+	"github.com/trisacrypto/directory/pkg/trtl/config"
+	"github.com/trisacrypto/directory/pkg/utils/bufconn"
+	"github.com/trisacrypto/directory/pkg/utils/logger"
+	"github.com/trisacrypto/trisa/pkg/ivms101"
+	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+const (
+	metaRegion = "tauceti"
+	metaOwner  = "taurian"
+	metaPID    = 8
+	bufSize    = 1024 * 1024
+)
+
+type trtlStoreTestSuite struct {
+	suite.Suite
+	tmpdb string
+	conf  *config.Config
+	trtl  *trtl.Server
+	grpc  *bufconn.GRPCListener
+}
+
+func (s *trtlStoreTestSuite) SetupSuite() {
+	require := s.Require()
+
+	var err error
+	s.tmpdb, err = ioutil.TempDir("../testdata", "db-*")
+	require.NoError(err)
+
+	conf := config.Config{
+		Maintenance: false,
+		BindAddr:    ":4436",
+		LogLevel:    logger.LevelDecoder(zerolog.DebugLevel),
+		ConsoleLog:  true,
+		Database: config.DatabaseConfig{
+			URL:           fmt.Sprintf("leveldb:///%s", s.tmpdb),
+			ReindexOnBoot: false,
+		},
+		Replica: config.ReplicaConfig{
+			Enabled: false, // Replica is tested in the replica package
+			PID:     metaPID,
+			Region:  metaRegion,
+			Name:    metaOwner,
+		},
+		MTLS: config.MTLSConfig{
+			Insecure: true,
+		},
+	}
+
+	// Mark config as processed
+	conf, err = conf.Mark()
+	require.NoError(err)
+	s.conf = &conf
+
+	trtl, err := trtl.New(*s.conf)
```

Comment:
This is super cool - we're just creating a mock `trtl` server without doing any networking. You can't do that with postgres!